t/24-worker-overall.t

```diff
@@ -28,11 +29,7 @@ $ENV{OPENQA_CONFIG} = "$FindBin::Bin/data/24-worker-overall";
 #       file specified via OPENQA_LOGFILE instead of stdout/stderr.
 $ENV{OPENQA_LOGFILE} = undef;
 
-# fake "/proc/loadavg"
-my $load_avg_file = tempfile('worker-overall-load-avg-XXXXX');
-my $load_avg_file_realpath = $load_avg_file->realpath;
```

Comment:
The `realpath` was only necessary because:
* `tempfile` by default creates a file in the current directory and saves it without the full path
* if you then go in a different directory and try to remove or modify it, it will fail
* To actually keep the file until the end of the test, the File::Temp object still needs to be around, so two variables were needed

By using `TMPDIR` it will use a file in the tmp directory and automatically have the full path. Only one variable needed.